web/src/i18n/locales/ja.json

```diff
@@ -99,8 +99,8 @@
         "enNameRequired": "ローマ字で入力してください",
         "fieldRequired": "{0}が必要です",
         "invalidForm": "フォームに記入してください",
-        "jaNameCharacters": "漢字\/かなでお名前を入力してください",
-        "jaNameRequired": "漢字\/かなで入力してください",
+        "jaNameCharacters": "漢字\/全角かなでお名前を入力してください",
+        "jaNameRequired": "漢字\/全角かなで入力してください",
```

Comment:
It appears the translation on this line has gotten out of sync with the english key. We should be translating "Japanese name is required" and don't need to mention the full-width or character types here.

Reply:
@ann-kilzer thanks, I have updated the message.